js/views/userListVw.js

```diff
@@ -26,11 +26,10 @@ module.exports = Backbone.View.extend({
 
   render: function(){
     var self = this;
-    this.listWrapper = $('<div class="flexRow flexExpand border0 custCol-border"></div>');
+    this.listWrapper = $('<div class="list flexRow flexExpand border0 custCol-border"></div>');
     if(this.usersShort.models.length > 0)
     {
-      __.each(this.usersShort.models, function (user)
-      {
+      this.usersShort.each(function (user) {
```

Comment:
It is using Underscore's each. this.userShort points to a Backbone.Collection which proxies a bunch of underscore methods right onto the Collection object.

http://backbonejs.org/#Collection-Underscore-Methods

It's essentially the same as what was happening before, but (arguably) a smoother syntax.